src/Microsoft.DotNet.Build.Tasks/PackageFiles/Build.Common.props

```diff
@@ -28,7 +28,7 @@
     <ToolsDir Condition="'$(ToolsDir)'==''">$(MSBuildThisFileDirectory)</ToolsDir>
     <DotnetCliPath Condition="'$(DotnetCliPath)'==''">$(ToolRuntimePath)dotnetcli/</DotnetCliPath>
     <OverrideToolHost>$(DotnetCliPath)dotnet</OverrideToolHost>
-    <BuildToolsTaskDir Condition="'$(BuildToolsTargets45)' == 'true'">$(ToolsDir)net45/</BuildToolsTaskDir>
+    <BuildToolsTaskDir Condition="'$(BuildToolsTargetsDesktop)' == 'true'">$(ToolsDir)net46/</BuildToolsTaskDir>
```

Comment:
As an FYI changing this to net46 is going to likely cause issues with the next set of folks picking up this change. If we find any people hitting that instead of fixing that spot I think we should remove it and have it use the settings from this common props.

Also it might be worth defaulting BuildToolsTargetsDesktop to true if BuildToolsTargets45 is set to help ease the transition. See https://github.com/dotnet/corefx/blob/master/dir.props#L30 and I assume the same is in coreclr/wcf repos.